frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1014,16 +1014,19 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                 {primaryRows.map((row, rowIndex) => (
                   <Tbody key={`${row.props.key}-tablebody`} isExpanded={row.isOpen}>
                     <Tr key={`${row.props.key}-tablerow`} ouiaId={row?.props?.key}>
-                      {onCollapse && addedSubRows[rowIndex] && (
-                        <Td
-                          expand={{
-                            isExpanded: row.isOpen || false,
-                            rowIndex,
-                            onToggle: onCollapse,
-                            expandId: 'expandable-toggle',
-                          }}
-                        />
-                      )}
+                      {onCollapse &&
+                        (addedSubRows[rowIndex] ? (
+                          <Td
+                            expand={{
+                              isExpanded: row.isOpen || false,
+                              rowIndex,
+                              onToggle: onCollapse,
+                              expandId: 'expandable-toggle',
+                            }}
+                          />
+                        ) : (
+                          <Td />
```

Comment:
- Always render a placeholder expand cell when the table is in expandable mode, even for rows without subrows (that do not render an expandable button).